### DISCUSSION TITLE
PVP Feature still in dev ?

### DISCUSSION THREAD

**Discussion by ZergHeureux** [ldtteam/minecolonies#9299] [Q&A]
Hi ! I'm a huge fan of the mod ! We play we some friends and i was wandering if y'all are still working on the P.V.P feature ? I saw some PR completed but not much about PVP in code, ingame or in the wiki ^^

**Comment by Talyda** [comment]
If you go on discord and go into #bot-commands channel and type !pvp it will tell you how it works right now (I am entirely too lazy to go there and copy paste right now =p)
Basically set each other as hostile, take your guards with you and go to it!

**Reply by ZergHeureux** [reply]
Thanks buddy ! :D
